src/java.sql/share/classes/java/sql/ResultSetMetaData.java

```diff
@@ -43,7 +43,8 @@
  *
  * @since 1.1
  */
-
+import org.checkerframework.checker.index.qual.NonNegative;
```

Comment:
Import statements should go with the `package` statement, not between the class Javadoc and the class definition.